kubernetes-azure-python/__main__.py

```diff
@@ -0,0 +1,120 @@
+"""An Azure RM Python Pulumi program"""
+
+import base64
+import pulumi
+from pulumi_azure_native import resources
+from pulumi_azure_native import network
+from pulumi_azure_native import containerservice
+
+# Get some values from the Pulumi configuration (or use defaults)
+azure_cfg = pulumi.Config("azure-native")
+azureLocation = azure_cfg.require("location")
+proj_cfg = pulumi.Config()
+num_worker_nodes = proj_cfg.get_float("numWorkerNodes", 3)
+k8s_version = proj_cfg.get("kubernetesVersion", "1.24.3")
+prefix_for_dns = proj_cfg.get("prefixForDns", "pulumi")
+node_vm_size = proj_cfg.get("nodeVmSize", "Standard_DS2_v2")
+mgmt_group_id = proj_cfg.require("mgmtGroupId")
+ssh_pub_key = proj_cfg.require("sshPubKey")
+
+# Create an Azure Resource Group
+resource_group = resources.ResourceGroup("resource_group",
```

Comment:
[PEP8](https://peps.python.org/pep-0008/#indentation) linters will flag all of these instances as not being aligned to PEP8. You have two options:
**Option A**:
```python
resource_group = resources.ResourceGroup("resource_group",
                                         location=azureLocation,
                                         resource_group_name="aks-rg"
                                         )
```

**Option B** (my personal preference because it reduces the line-too-long errors that will happen from other linters):
```python
resource_group = resources.ResourceGroup(
    "resource_group",
    location=azureLocation,
    resource_group_name="aks-rg"
)
```

Reply:
Switched to Option B in c4dea65, should be resolved now

Reply:
Hm, not quite right. I'll make suggestions.